src/TextBox.js

```diff
@@ -226,7 +226,7 @@ export default class TextBox extends BaseClass {
     }
 
     function rotate(text) {
-      text.attr("transform", (d, i) => `rotate(${that._rotate(d, i)}, ${d.x + d.w / 2}, ${d.y + d.h / 2})translate(${d.x}, ${d.y})`);
+      text.attr("transform", (d, i) => `translate(${d.x}, ${d.y}) rotate(${that._rotate(d, i)}, ${that._rotationAnchor && that._rotationAnchor(d, i) ? that._rotationAnchor(d, i)[0] : d.w / 2}, ${that._rotationAnchor && that._rotationAnchor(d, i) ? that._rotationAnchor(d, i)[1] : d.h / 2})`);
```

Comment:
This line is gnarly long. Can you make the default value of the rotation anchor do the `d.x / 2` logic, and then fetch the array as a variable on the previous line?